addon/lib/field.js

```diff
@@ -234,9 +234,17 @@ export default EmberObject.extend(Evented, {
    * @method validate
    */
   validate: task(function*() {
+    const specificValidation = this.get(`_validate${this.question.__typename}`);
+    if (!specificValidation) {
+      // eslint-disable-next-line no-console
+      console.error(
+        "Missing validation function for",
+        this.question.__typename
+      );
+    }
```

Comment:
Use `assert` from `@ember/debug` for those kind of logs / errors

Reply:
Fixed. Still have to figure out why the test is failing ...

Reply:
Found it, was a curly brace ...